test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPreserveCompilationContext.cs

```diff
@@ -286,9 +346,5 @@ public void It_publishes_the_project_with_a_refs_folder_and_correct_deps_file()
 System.Xml.ReaderWriter.dll
 System.Xml.XDocument.dll
 TestLibrary.dll".Split(new char[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries);
-
-        public GivenThatWeWantToPreserveCompilationContext(ITestOutputHelper log) : base(log)
-        {
-        }
```

Comment:
Thanks for moving this. I generated these with (Ctrl + '.'). I don't know why this one didn't go at the top like the others.